includes/utils.php

```diff
@@ -1178,7 +1178,7 @@ function update_content_image_urls( int $post_id, array $images ) {
 	/**
 	 * Filter whether image URLS should be updated in the content.
 	 *
-	 * @since x.x.x
+	 * @since 2.1.0
```

Comment:
nice catch, may be worth adding to the release steps as well?

Reply:
It is already there: `Since tag updates: ensure `@since` tags indicate the new version, replacing `x.x.x`, `n.e.x.t` and other placeholders`

Though I don't think we have this same step across our other plugins so would be nice to add that in, as we often don't update those until I see them later